### DISCUSSION TITLE
Are their any exploits for chrome://settings?

### DISCUSSION THREAD

**Discussion by Charlieis1nsan3** [3kh0/ext-remover#1370] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I dont know any and Im on v130

**Comment by ghost** [comment]
nope

**Comment by ghost** [comment]
except corrupting extension wanna learn how?

**Reply by Charlieis1nsan3** [reply]
theirs crimson, I know.

**Comment by schoolexploitkid** [comment]
there's corkey (image go poof bc I forgor to censor)

**Reply by schoolexploitkid** [reply]
heres the censored img: 
![image](https://github.com/user-attachments/assets/3caca7bb-485d-47bd-b349-b7f00207c16a)

**Reply by Charlieis1nsan3** [reply]
yeah that's kinda the thing, I don't have access to chrome://extensions. but I want an exploit. ( like Crimson)

**Comment by funkymokey** [comment]
Everyone in this repository should try shimboot. Like genuinely. You wont have to go through the garbage methods to exploit on Chrome OS, and instead just run Debian off a USB.

**Reply by Charlieis1nsan3** [reply]
yeah I know shimboot, but I want something else.

**Reply by S-PScripts** [reply]
❎ ❎ ❎ 

https://github.com/search?q=repo%3AS-PScripts%2Fchromebook-utilities%20chrome%3A%2F%2Fsettings&type=code

**Reply by ghost** [reply]
For corkey you only need it to see if it the extension says corrupted so you possibly no need chrome://extensions to be unblocked.
